### PR TITLE
qtscrcpy: Add version 1.4.5

### PR DESCRIPTION
**Pull request by Rehtt**

### CODE REVIEW

File: bucket/qtscrcpy.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "version":"1.4.5",
+    "description": "QtScrcpy connects to Android devices via USB (or via TCP/IP) for display and control. It does NOT require the root privileges",
+    "homepage": "https://github.com/barry-ran/QtScrcpy",
+    "license":"Apache Licence 2.0",
```
Comment (Ash258) [inline bucket/qtscrcpy.json:5]: Invalid SPDX identifier

File: bucket/qtscrcpy.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/barry-ran/QtScrcpy/releases/download/v1.4.5/QtScrcpy-win-x64-v1.4.5.zip",
+            "extract_dir":"QtScrcpy-win-x64-v1.4.5"
+        },
+        "32bit":{
+            "url":"https://github.com/barry-ran/QtScrcpy/releases/download/v1.4.5/QtScrcpy-win-x86-v1.4.5.zip",
+            "extract_dir":"QtScrcpy-win-x86-v1.4.5"
```
Comment (Ash258) [inline bucket/qtscrcpy.json:13]: Missing hashes

File: bucket/qtscrcpy.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "bin":"QtScrcpy.exe",
+    "shortcuts": [
+        [
+            "QtScrcpy.exe",
+            "QtScrcpy"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/barry-ran/QtScrcpy"
+    },
```
Comment (Ash258) [inline bucket/qtscrcpy.json:25]: simplify `checkver: github`

File: bucket/qtscrcpy.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "autoupdate": {
+        "64bit": {
+            "url": "https://github.com/barry-ran/QtScrcpy/releases/download/v$version/QtScrcpy-win-x64-v$version.zip",
+            "extract_dir":"QtScrcpy-win-x64-v$version"
+        },
+        "32bit":{
+            "url":"https://github.com/barry-ran/QtScrcpy/releases/download/v$version/QtScrcpy-win-x86-v$version.zip",
+            "extract_dir":"QtScrcpy-win-x86-v$version"
+        }
```
Comment (Ash258) [inline bucket/qtscrcpy.json:34]: missing architecture

File: bucket/qtscrcpy.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
+{
+    "version":"1.4.5",
+    "description": "QtScrcpy connects to Android devices via USB (or via TCP/IP) for display and control. It does NOT require the root privileges",
```
Comment (Ash258) [inline bucket/qtscrcpy.json:3]: do not mention application name in description

**Review by Ash258 — CHANGES_REQUESTED** [review]
HOw is this different from https://github.com/ScoopInstaller/Main/blob/master/bucket/scrcpy.json??

**Comment by Rehtt** [comment]
The difference between QtScrcpy and the original scrcpy is as follows:

keys|scrcpy|QtScrcpy
--|:--:|:--:
ui|sdl|qt
video encode|ffmpeg|ffmpeg
video render|sdl|opengl
cross-platform|self implemented|provided by Qt
language|C|C++
style|sync|async
keymap|no custom keymap|support custom keymap
build|meson+gradle|Qt Creator

File: bucket/qtscrcpy.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        },
+        "32bit": {
+            "url": "https://github.com/barry-ran/QtScrcpy/releases/download/v$version/QtScrcpy-win-x86-v$version.zip",
+            "extract_dir": "QtScrcpy-win-x86-v$version"
+        }
+        },
+        "hash": {
+            "url": "$baseurl/SHA256SUMS.txt"
+        }
+
```
Comment (Ash258) [inline bucket/qtscrcpy.json:40]: There is no such file in the releases

File: bucket/qtscrcpy.json [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/barry-ran/QtScrcpy/releases/download/v1.4.5/QtScrcpy-win-x64-v1.4.5.zip",
+            "extract_dir": "QtScrcpy-win-x64-v1.4.5",
+            "hash": "644d11306c2fc3f9ee7f70051005c2d576bce3aa5013f7a9ff61da31e97223e4"
+        },
+        "32bit": {
+            "url": "https://github.com/barry-ran/QtScrcpy/releases/download/v1.4.5/QtScrcpy-win-x86-v1.4.5.zip",
+            "extract_dir": "QtScrcpy-win-x86-v1.4.5",
+            "hash": "c6806f188ca21bd7ead9519eacd3b15eef5df6fb9d9dd5766ba74310a4b4cc25"
```
Comment (Ash258) [inline bucket/qtscrcpy.json:15]: Hashes before extract_dir

**Review by Ash258 — CHANGES_REQUESTED** [review]
Execute checkver and push the formated manifest

File: bucket/qtscrcpy.json [hunk 8]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "1.4.5",
+    "description": "Connects to Android devices via USB (or via TCP/IP) for display and control.",
```
Comment (Ash258) [inline bucket/qtscrcpy.json:3]:
```suggestion
    "description": "Display and control Android device via TCP/IP or USB",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Should be `config` folder persisted? What everything is stored there?

A

**Comment by Rehtt** [comment]
No need to keep the `config` folder

**Comment by Ash258** [comment]
... A

**Comment by Rehtt** [comment]
Oh, that's a user profile, I rarely use it so I forgot

**Comment by 7wells** [comment]
Hello, is somebody still working on this?

A happy New Year to all of you!

**Pull request merged**